react-exercises/modal-react/src/Components/Button/styles.scss

```
@@ -0,0 +1,37 @@
+@import "../../sass/global";
+
+.btn{
+    display: flex;
+    max-width: 10rem;
+    max-height: 5rem;
+    color: white;
+    cursor: pointer;
+}
+
+.btn-primary{
```

please make use of sass features 
you can put .btn-primary inside .btn 
like 
```suggestion
.btn {
&.btn-primary {}
}
```